{"name": "test_login", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not callable", "trace": "self = <tests.test_login.TestLogin object at 0x7a598eefb390>\ninitialize_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a93f8ba7f0af2ab6e8417b062768c855\")>\nlogger_creation = None\n\n    def test_login(self, initialize_driver,logger_creation):\n      home_page = HomePage(initialize_driver)\n>     logger_creation()\nE     TypeError: 'NoneType' object is not callable\n\ntests/test_login.py:18: TypeError"}, "start": 1732029249252, "stop": 1732029249253, "uuid": "77461bcc-aa73-44e9-bee6-9fcbd23a9b86", "historyId": "2116f0ad00ab1de0effc54f114dd9496", "testCaseId": "2116f0ad00ab1de0effc54f114dd9496", "fullName": "tests.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "aelita-hp-pavilion"}, {"name": "thread", "value": "48191-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}